TriblerGUI/widgets/triblertablecontrollers.py

```diff
@@ -85,15 +85,21 @@ def perform_query(self, **kwargs):
                                          self.on_query_results,
                                          url_params=kwargs)
 
-    def on_query_results(self, response):
+    def on_query_results(self, response, remote=False):
```

Comment:
Could you add a (docstring) comment here what the impact of the `remote` parameter exactly is?